test/functional/test_intermediate_output.py

```diff
@@ -31,7 +31,7 @@
 
 
 def _timestamp():
-    return time.strftime("%Y-%m-%d-%H-%M-%S".format(time.gmtime(time.time())))
+    return time.strftime("%Y-%m-%d-%H-%M-%S".format(time.gmtime(time.time())))  # noqa: F523
```

Comment:
```suggestion
    return time.strftime("%Y-%m-%d-%H-%M-%S")
```

I'd rather we fix the code rather than putting "noqa" when possible. I *think* the `format` call is doing nothing here?

```python
>>> time.strftime("%Y-%m-%d-%H-%M-%S".format(time.gmtime(time.time())))
'2020-05-13-13-25-11'
>>> time.strftime("%Y-%m-%d-%H-%M-%S")
'2020-05-13-13-25-13'
```